corehq/apps/smsbillables/models.py

```diff
@@ -312,26 +321,40 @@ def create(cls, message_log, api_response=None):
         country_code, national_number = get_country_code_and_national_number(phone_number)
 
         if backend_instance is None or _sms_backend_is_global(backend_instance):
-            billable.gateway_fee = SmsGatewayFee.get_by_criteria(
-                backend_api_id,
-                direction,
-                backend_instance=backend_instance,
-                country_code=country_code,
-                national_number=national_number,
-            )
-            if billable.gateway_fee is not None:
-                conversion_rate = billable.gateway_fee.currency.rate_to_default
+            if backend_instance is None and backend_api_id == TwilioBackend.get_api_id():
+                def _get_twilio_client():
+                    twilio_backend = MobileBackend.load_by_name(None, "MOBILE_BACKEND_TWILIO")
+                    account_sid = twilio_backend.account_sid
+                    auth_token = twilio_backend.auth_token
+                    return TwilioRestClient(account_sid, auth_token)
+
+                twilio_message = _get_twilio_client().messages.get(message_log.backend_message_id)
```

Comment:
we should make sure that we properly handle when the message id is not found or if any error happens here in this lookup